android/expoview/src/main/java/versioned/host/exp/exponent/modules/api/LocationModule.java

```diff
@@ -344,6 +365,70 @@ public void removeWatchAsync(final int watchId, final Promise promise) {
     promise.resolve(null);
   }
 
+  @ReactMethod
+  public void geocodeAsync(final String address, final Promise promise) {
+    if (isMissingPermissions()) {
+      promise.reject("E_LOCATION_UNAUTHORIZED", "Not authorized to use location services");
+      return;
+    }
+
+    if (Geocoder.isPresent()) {
+    SmartLocation.with(mScopedContext).geocoding()
```

Comment:
Nope, it doesn't. It uses only the Android one and if the Geocoder is not present an error is thrown with a message `Please check if Geocoder.isPresent() before invoking the search`